packages/synthetix-main/cannonfile.test.toml

```diff
@@ -11,6 +11,13 @@ defaultValue = "main"
 description = "Admin user for the system"
 defaultValue = "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266"
 
+[setting.oracle_manager_version]
+defaultValue = "1.0.0"
```

Comment:
this should be `[setting.oracle_manager_source]` to emcompass the entire oracle manager source in case its package is coming from IPFS